catalog/ui/catalog-ui-search/src/main/java/org/codice/ddf/catalog/ui/config/ConfigurationApplication.java

```diff
@@ -169,6 +169,8 @@
 
   private int maximumUploadSize = 1_048_576;
 
+  private int maximumNumListItems = 3;
```

Comment:
✏️ `listItemLimit`

Reply:
this limit seem low. What if some one has more than 3 list item already. They will permanently unable to load or view their existing list.

Reply:
I would recommend a higher number, such as the page size (250)